api/ably-token-request/index.ts

```diff
@@ -3,24 +3,18 @@ import { Context, HttpRequest } from "@azure/functions";
 import { authorized, ApiRequestContext } from "../common/ApiRequestContext";
 import * as Ably from "ably/promises";
 
-const client = new Ably.Realtime(process.env.ABLY_API_KEY);
+const client = new Ably.Rest(process.env.ABLY_API_KEY);
 
-export default async function (
-  context: Context,
-  req: HttpRequest
-): Promise<void> {
+export default async function (context: Context, req: HttpRequest): Promise<void> {
   await authorized(
     context,
     req,
     async ({ user }: ApiRequestContext) => {
-      const clientId = `${user.id}:${user.username}`;
-
-      const tokenRequestData = await client.auth.createTokenRequest({
-        clientId,
-      });
+      const clientId = `${user.id}:${user.username}:${encodeURIComponent(user.profileImgUrl)}`;
```

Comment:
We probably shouldn't embed the profile image in the client ID, but we'll update this in a future PR, possibly also to remove the username. (This was me setting a bad example here! Sorry!)
These should probably be fetched from a user profile API and cached in the browser.